lib/rules/pseudo-element.js

```diff
@@ -1,52 +1,19 @@
 'use strict';
 
-var helpers = require('../helpers');
-
-var pseudoElements = [
-  'after',
-  'before',
-  'first-letter',
-  'first-line',
-  'selection',
-  'backdrop'
-];
-
-var pseudoClasses = [
-  'active',
-  'checked',
-  'disabled',
-  'empty',
-  'enabled',
-  'first-child',
-  'first-of-type',
-  'focus',
-  'hover',
-  'in-range',
-  'invalid',
-  'lang',
-  'last-child',
-  'last-of-type',
-  'link',
-  'not',
-  'nth-child',
-  'nth-last-child',
-  'nth-last-of-type',
-  'nth-of-type',
-  'only-of-type',
-  'only-child',
-  'optional',
-  'out-of-range',
-  'read-only',
-  'read-write',
-  'required',
-  'root',
-  'target',
-  'valid',
-  'visited'
-];
+var helpers = require('../helpers'),
+    yaml = require('js-yaml'),
+    fs = require('fs'),
+    path = require('path');
+
+var pseudoElements = yaml.safeLoad(
+      fs.readFileSync(path.join(__dirname, '../../data', 'pseudoElements.yml'), 'utf8')
```

Comment:
Do we really need these in separate files right now? They're not being used anywhere else and with such small datasets it seems unnecessary to force two file read operations on users.